about.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/2649a194d9.js" crossorigin="anonymous"></script>
+</head>
+<body>
+
+    <div class="dskNav1">
+        <ul class="dskNavUl1">
+            <li class="nav-item">
+                <a href="#" class="nav-link"><i class="fa-brands fa-facebook-f"></i></a>
+            </li>
+            <li class="nav-item">
+                <a href="#" class="nav-link"><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li><a href="#">English</li></a>
+            <li><a href="#">My Page</li></a>
+            <li><a href="#">Logout</li></a>
+        </ul>
+
+        <div class="dskNav2">
+            <div class="logoImg">
+                <img src="./images/myLogo.jpg" alt="Logo">
+                <p>Adorable Forex Trading</p>
+            </div>
+
+            <ul class="dskNavUl2">               
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="#mainProgram">Program</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#speakerContent">Speakers</a></li>
+                <li><a href="#footerLogo">Partners</a></li>
+                <li class="classes"><a href="#footerLogo">Classes</a></li>
+            </ul>           
+        </div>
+    </div>
```

Comment:
- [ ]  Please wrap this section in a header element. This is more semantically correct.